src/main/java/uk/gov/pay/api/filter/ratelimit/RateLimiter.java

```diff
@@ -12,50 +13,51 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(RateLimiter.class);
 
-    private final int rate;
-    private final int rateForPost;
+    private int rate;
+    private int rateForPost;
+    private int auditRate;
+    private int perMillis;
```

Comment:
is this requests per millis? it would be better to be more explicit, e.g. `requestPerMillis`

Reply:
It is rate per millis.

Reply:
we always set rate as millis and usually set to 1000 ~ 1 second. So it will effectively be "x requests per second". This allows us to reduce the granularity to milli seconds if required.